### ISSUE TITLE
web app on Master-v34 setup issues on Archlinux/EOS

### ISSUE THREAD

**Issue by jfabernathy** [MythTV/mythtv#817]
* **Platform**:Endeavour OS (Archlinux)

* **MythTV version**:Master v34

* **Package version**: built from AUR mythtv-git

* **Component**:setup

### What steps will reproduce the bug?
initial setup on clean OS install

### Additional information

I'm trying to come up with a repeatable initial setup procedure using the Web App for installing mythtv on an Archlinux system, specifically EndeavourOS. I'm  using the AUR package mythtv-git and the command yay -S mythtv-git --editmenu   to build.  During the edit, I change from v33 to master.

The build is completes okay.

I setup mariadb with:
```
sudo pacman -S --needed mariadb
sudo mariadb-install-db --user=mysql --basedir=/usr --datadir=/var/lib/mysql
sudo systemctl enable --now mariadb.service
sudo mysql_secure_installation

#setup timezone info
mysql_tzinfo_to_sql /usr/share/zoneinfo | sudo mysql -uroot mysql

sudo systemctl restart mysql
```
Next I open a console and run mythbackend.  In a browser I go to localhost:6544.  This starts the web app.

The instructions for setting up mythconverg popup and I select mariadb and copy the text to another console session to create the setup.sql per the screen instructions. Then I run then mysql command to import the setup.sql and that works.

Following the screen instructions I restart the backend via the button in web app.  This connection to the database is okay, but the middle indicator is still getting an error.

I tried restarted the backend via the button and uses the save button.  I tried this several times and finally I could see the mythbackend console show conversion of the schema versions.  At that point I can save and go to the next screen which is the language screen.

below is the mythbackend console log:
```
[jim@eos-mythtv ~]$ mythbackend
2023-11-19 20:55:21.988060 C  mythbackend version: makepkg [v34-Pre-544-g4908066fc4] www.mythtv.org
2023-11-19 20:55:21.988064 C  Qt version: compile: 5.15.11, runtime: 5.15.11
2023-11-19 20:55:21.988082 I  EndeavourOS (x86_64)
2023-11-19 20:55:21.988083 N  Enabled verbose msgs:  general
2023-11-19 20:55:21.988086 N  Setting Log Level to LOG_INFO
2023-11-19 20:55:21.998545 I  Added logging to the console
2023-11-19 20:55:21.998662 I  Setup Interrupt handler
2023-11-19 20:55:21.998664 I  Setup Terminated handler
2023-11-19 20:55:21.998666 I  Setup Segmentation fault handler
2023-11-19 20:55:21.998667 I  Setup Aborted handler
2023-11-19 20:55:21.998669 I  Setup Floating point exception handler
2023-11-19 20:55:21.998671 I  Setup Illegal instruction handler
2023-11-19 20:55:21.998672 I  Setup Bus error handler
2023-11-19 20:55:21.998676 I  Setup Real-time signal 0 handler
2023-11-19 20:55:21.998679 I  Setup Hangup handler
2023-11-19 20:55:21.999327 N  Using runtime prefix = /usr
2023-11-19 20:55:21.999328 N  Using configuration directory = /home/jim/.mythtv
2023-11-19 20:55:21.999382 I  Assumed character encoding: en_US.UTF-8
2023-11-19 20:55:21.999429 N  Setting QT default locale to en_US
2023-11-19 20:55:21.999430 I  Current locale en_US
2023-11-19 20:55:22.000145 N  Reading locale defaults from /usr/share/mythtv//locales/en_us.xml
2023-11-19 20:55:22.026574 I  Power: On AC power
2023-11-19 20:55:22.026581 I  Power: Supported actions: Suspend,Restart,Shutdown
2023-11-19 20:55:22.027781 I  Loading en_us translation for module mythfrontend
2023-11-19 20:55:22.031914 N  ***********************************************************************
2023-11-19 20:55:22.031915 N  ***** MythBackend starting in Web App only mode for initial setup *****
2023-11-19 20:55:22.031915 N  ***** Use http://<yourBackend>:6544 to perform setup              *****
2023-11-19 20:55:22.031916 N  ***********************************************************************
2023-11-19 20:55:22.032061 I  HTTPPool: Using maximum 12 threads
2023-11-19 20:55:22.047963 W  SSL: SSL Host key file (/home/jim/.mythtv/certificates/key.pem) does not exist or is not readable
2023-11-19 20:55:22.048084 I  Listening on TCP 0.0.0.0:6544
2023-11-19 20:55:22.048106 I  Listening on TCP [::]:6544
2023-11-19 20:55:22.048117 I  Listening on TCP 0.0.0.0:6744
2023-11-19 20:55:22.048128 I  Listening on TCP [::]:6744
2023-11-19 20:55:23.009163 I  Bonjour: Service registration complete: name 'mythbackend on eos-mythtv' type '_http._tcp.' domain: 'local.'
2023-11-19 20:55:29.065438 I  Service '/services/' introspection complete
2023-11-19 20:55:29.331488 I  Service 'Myth' introspection complete
2023-11-19 20:55:29.332415 I  Service 'Config' introspection complete
2023-11-19 20:55:29.336650 E  MySQL time zone support check failed
2023-11-19 20:55:29.423599 E  MySQL time zone support check failed
2023-11-19 20:55:29.429466 E  [dbtest] Unable to connect to database!
2023-11-19 20:55:29.429473 E  Driver error was [1/1698]:
QMYSQL: Unable to connect
Database error was:
Access denied for user 'mythtv'@'localhost'

2023-11-19 20:55:29.434274 I  Qt: QSqlDatabasePrivate::addDatabase: duplicate connection name 'dbtest', old connection removed.
2023-11-19 20:55:29.436042 E  [dbtest] Unable to connect to database!
2023-11-19 20:55:29.436049 E  Driver error was [1/1698]:
QMYSQL: Unable to connect
Database error was:
Access denied for user 'mythtv'@'localhost'

2023-11-19 20:55:29.515444 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:55:29.515716 I  Qt: QSqlDatabasePrivate::addDatabase: duplicate connection name 'dbtest', old connection removed.
2023-11-19 20:55:29.516737 E  Service Exception: Database Error executing query.
2023-11-19 20:55:29.521591 E  [dbtest] Unable to connect to database!
2023-11-19 20:55:29.521598 E  Driver error was [1/1698]:
QMYSQL: Unable to connect
Database error was:
Access denied for user 'mythtv'@'localhost'

2023-11-19 20:55:29.554902 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:55:29.554926 E  Service Exception: Database Error executing query.
2023-11-19 20:55:29.560175 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:55:29.560206 E  Service Exception: Database Error executing query.
2023-11-19 20:55:29.564789 I  Service 'Dvr' introspection complete
2023-11-19 20:55:29.565043 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:55:29.565065 E  Service Exception: Database Error executing query.
2023-11-19 20:55:29.567322 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:55:29.567345 E  Service Exception: Database Error executing query.
2023-11-19 20:55:42.490726 I  Qt: QSqlDatabasePrivate::addDatabase: duplicate connection name 'dbtest', old connection removed.
2023-11-19 20:55:42.491337 E  [dbtest] Unable to connect to database!
2023-11-19 20:55:42.491345 E  Driver error was [1/1698]:
QMYSQL: Unable to connect
Database error was:
Access denied for user 'mythtv'@'localhost'

2023-11-19 20:57:29.464621 E  MySQL time zone support check failed
2023-11-19 20:57:29.468965 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:57:29.468989 E  Service Exception: Database Error executing query.
2023-11-19 20:57:29.472138 I  Qt: QSqlDatabasePrivate::addDatabase: duplicate connection name 'dbtest', old connection removed.
2023-11-19 20:57:29.490445 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:57:29.490480 E  Service Exception: Database Error executing query.
2023-11-19 20:57:29.494784 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:57:29.494807 E  Service Exception: Database Error executing query.
2023-11-19 20:57:30.256710 C  mythbackend version: makepkg [v34-Pre-544-g4908066fc4] www.mythtv.org
2023-11-19 20:57:30.256718 C  Qt version: compile: 5.15.11, runtime: 5.15.11
2023-11-19 20:57:30.256753 I  EndeavourOS (x86_64)
2023-11-19 20:57:30.256755 N  Enabled verbose msgs:  general
2023-11-19 20:57:30.256759 N  Setting Log Level to LOG_INFO
2023-11-19 20:57:30.267472 I  Added logging to the console
2023-11-19 20:57:30.267625 I  Setup Interrupt handler
2023-11-19 20:57:30.267628 I  Setup Terminated handler
2023-11-19 20:57:30.267630 I  Setup Segmentation fault handler
2023-11-19 20:57:30.267632 I  Setup Aborted handler
2023-11-19 20:57:30.267635 I  Setup Floating point exception handler
2023-11-19 20:57:30.267637 I  Setup Illegal instruction handler
2023-11-19 20:57:30.267639 I  Setup Bus error handler
2023-11-19 20:57:30.267644 I  Setup Real-time signal 0 handler
2023-11-19 20:57:30.267648 I  Setup Hangup handler
2023-11-19 20:57:30.267739 N  Using runtime prefix = /usr
2023-11-19 20:57:30.267740 N  Using configuration directory = /home/jim/.mythtv
2023-11-19 20:57:30.267829 I  Assumed character encoding: en_US.UTF-8
2023-11-19 20:57:30.267900 N  Setting QT default locale to en_US
2023-11-19 20:57:30.267901 I  Current locale en_US
2023-11-19 20:57:30.267918 N  Reading locale defaults from /usr/share/mythtv//locales/en_us.xml
2023-11-19 20:57:30.318731 I  Power: On AC power
2023-11-19 20:57:30.318740 I  Power: Supported actions: Suspend,Restart,Shutdown
2023-11-19 20:57:30.318823 I  Loading en_us translation for module mythfrontend
2023-11-19 20:57:30.326856 N  ***********************************************************************
2023-11-19 20:57:30.326858 N  ***** MythBackend starting in Web App only mode for initial setup *****
2023-11-19 20:57:30.326859 N  ***** Use http://<yourBackend>:6544 to perform setup              *****
2023-11-19 20:57:30.326860 N  ***********************************************************************
2023-11-19 20:57:30.326909 I  HTTPPool: Using maximum 12 threads
2023-11-19 20:57:30.338727 W  SSL: SSL Host key file (/home/jim/.mythtv/certificates/key.pem) does not exist or is not readable
2023-11-19 20:57:30.338806 I  Listening on TCP 0.0.0.0:6544
2023-11-19 20:57:30.338829 I  Listening on TCP [::]:6544
2023-11-19 20:57:30.338844 I  Listening on TCP 0.0.0.0:6744
2023-11-19 20:57:30.338929 I  Listening on TCP [::]:6744
2023-11-19 20:57:31.173317 I  Bonjour: Service registration complete: name 'mythbackend on eos-mythtv' type '_http._tcp.' domain: 'local.'
2023-11-19 20:57:32.175114 I  Service '/services/' introspection complete
2023-11-19 20:57:32.175460 I  Service 'Myth' introspection complete
2023-11-19 20:57:32.180207 E  MySQL time zone support check failed
2023-11-19 20:57:32.184964 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:57:32.185028 E  Service Exception: Database Error executing query.
2023-11-19 20:57:32.187329 I  Service 'Config' introspection complete
2023-11-19 20:57:32.204123 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:57:32.204147 E  Service Exception: Database Error executing query.
2023-11-19 20:57:32.206961 E  DB Error (API Myth/GetSetting ):
Query was:

Driver error was [1/]:
Driver not loaded
Database error was:
Driver not loaded

2023-11-19 20:57:32.207159 E  Service Exception: Database Error executing query.
2023-11-19 20:57:37.280075 I  Service 'Dvr' introspection complete
2023-11-19 20:58:15.906171 N  MythBackend Web App only mode exiting
2023-11-19 20:58:15.906469 I  Bonjour: De-registering service '_http._tcp.' on 'mythbackend on eos-mythtv'
2023-11-19 20:58:15.907909 I  Restarting mythbackend
2023-11-19 20:58:16.044586 C  mythbackend version: makepkg [v34-Pre-544-g4908066fc4] www.mythtv.org
2023-11-19 20:58:16.044594 C  Qt version: compile: 5.15.11, runtime: 5.15.11
2023-11-19 20:58:16.044629 I  EndeavourOS (x86_64)
2023-11-19 20:58:16.044630 N  Enabled verbose msgs:  general
2023-11-19 20:58:16.044634 N  Setting Log Level to LOG_INFO
2023-11-19 20:58:16.055494 I  Added logging to the console
2023-11-19 20:58:16.055657 I  Setup Interrupt handler
2023-11-19 20:58:16.055660 I  Setup Terminated handler
2023-11-19 20:58:16.055663 I  Setup Segmentation fault handler
2023-11-19 20:58:16.055665 I  Setup Aborted handler
2023-11-19 20:58:16.055668 I  Setup Floating point exception handler
2023-11-19 20:58:16.055670 I  Setup Illegal instruction handler
2023-11-19 20:58:16.055672 I  Setup Bus error handler
2023-11-19 20:58:16.055677 I  Setup Real-time signal 0 handler
2023-11-19 20:58:16.055682 I  Setup Hangup handler
2023-11-19 20:58:16.055764 N  Using runtime prefix = /usr
2023-11-19 20:58:16.055765 N  Using configuration directory = /home/jim/.mythtv
2023-11-19 20:58:16.055865 I  Assumed character encoding: en_US.UTF-8
2023-11-19 20:58:16.056094 I  Empty LocalHostName. This is typical.
2023-11-19 20:58:16.056098 I  Using a profile name of: 'eos-mythtv' (Usually the same as this host's name.)
2023-11-19 20:58:16.056500 I  Start up testing connections. DB localhost, BE , attempt 0, status dbAwake, Delay: 2000
2023-11-19 20:58:17.073305 N  Setting QT default locale to en_US
2023-11-19 20:58:17.073310 I  Current locale en_US
2023-11-19 20:58:17.073367 N  Reading locale defaults from /usr/share/mythtv//locales/en_us.xml
2023-11-19 20:58:17.073638 E  SaveSettingOnHost('Country') - No database yet
2023-11-19 20:58:17.073641 E  SaveSettingOnHost('FreqTable') - No database yet
2023-11-19 20:58:17.073643 E  SaveSettingOnHost('ISO639Language0') - No database yet
2023-11-19 20:58:17.073646 E  SaveSettingOnHost('ISO639Language1') - No database yet
2023-11-19 20:58:17.073648 E  SaveSettingOnHost('Language') - No database yet
2023-11-19 20:58:17.073650 E  SaveSettingOnHost('TVFormat') - No database yet
2023-11-19 20:58:17.073652 E  SaveSettingOnHost('VbiFormat') - No database yet
2023-11-19 20:58:17.073658 E  SaveSettingOnHost('DateFormat') - No database yet
2023-11-19 20:58:17.073662 E  SaveSettingOnHost('MythArchiveDateFormat') - No database yet
2023-11-19 20:58:17.073665 E  SaveSettingOnHost('MythArchiveTimeFormat') - No database yet
2023-11-19 20:58:17.073669 E  SaveSettingOnHost('MythArchiveVideoFormat') - No database yet
2023-11-19 20:58:17.073672 E  SaveSettingOnHost('ShortDateFormat') - No database yet
2023-11-19 20:58:17.073675 E  SaveSettingOnHost('TimeFormat') - No database yet
2023-11-19 20:58:17.122205 I  Power: On AC power
2023-11-19 20:58:17.122219 I  Power: Supported actions: Suspend,Restart,Shutdown
2023-11-19 20:58:17.122334 I  Loading en_us translation for module mythfrontend
2023-11-19 20:58:17.131413 E  Error preparing query: SELECT data FROM settings WHERE value = :VALUE AND hostname = :HOSTNAME
2023-11-19 20:58:17.131419 E  Driver error was [2/1146]:
QMYSQL3: Unable to prepare statement
Database error was:
Table 'mythconverg.settings' doesn't exist

2023-11-19 20:58:17.131590 E  Error preparing query: SELECT data FROM settings WHERE value = :VALUE AND hostname = :HOSTNAME
2023-11-19 20:58:17.131593 E  Driver error was [2/1146]:
QMYSQL3: Unable to prepare statement
Database error was:
Table 'mythconverg.settings' doesn't exist

2023-11-19 20:58:17.135531 I  No current database version?
2023-11-19 20:58:17.136396 I  Database appears to be empty/new!
2023-11-19 20:58:17.137953 W  Not allowed to upgrade the database.
2023-11-19 20:58:17.139512 N  Inserting MythTV initial database information.
2023-11-19 20:58:17.139641 C  Upgrading to MythTV schema version 1307
2023-11-19 20:58:18.052412 C  Upgrading to MythTV schema version 1308
2023-11-19 20:58:18.075724 C  Upgrading to MythTV schema version 1309
2023-11-19 20:58:18.076719 C  Upgrading to MythTV schema version 1310
2023-11-19 20:58:18.077221 C  Upgrading to MythTV schema version 1311
2023-11-19 20:58:18.104500 C  Upgrading to MythTV schema version 1312
2023-11-19 20:58:18.115718 C  Upgrading to MythTV schema version 1313
2023-11-19 20:58:18.139392 C  Upgrading to MythTV schema version 1314
2023-11-19 20:58:18.140398 C  Upgrading to MythTV schema version 1315
2023-11-19 20:58:18.141040 C  Upgrading to MythTV schema version 1316
2023-11-19 20:58:18.189588 C  Upgrading to MythTV schema version 1317
2023-11-19 20:58:18.407282 C  Upgrading to MythTV schema version 1318
2023-11-19 20:58:18.427765 C  Upgrading to MythTV schema version 1319
2023-11-19 20:58:18.467127 C  Upgrading to MythTV schema version 1320
2023-11-19 20:58:18.504567 C  Upgrading to MythTV schema version 1321
2023-11-19 20:58:18.576798 C  Upgrading to MythTV schema version 1322
2023-11-19 20:58:18.597550 C  Upgrading to MythTV schema version 1323
2023-11-19 20:58:18.644762 C  Upgrading to MythTV schema version 1324
2023-11-19 20:58:18.665955 C  Upgrading to MythTV schema version 1325
2023-11-19 20:58:18.691092 C  Upgrading to MythTV schema version 1326
2023-11-19 20:58:18.710250 C  Upgrading to MythTV schema version 1327
2023-11-19 20:58:18.710717 C  Upgrading to MythTV schema version 1328
2023-11-19 20:58:18.753366 C  Upgrading to MythTV schema version 1329
2023-11-19 20:58:18.839892 C  Upgrading to MythTV schema version 1330
2023-11-19 20:58:18.883213 C  Upgrading to MythTV schema version 1332
2023-11-19 20:58:18.907592 C  Upgrading to MythTV schema version 1333
2023-11-19 20:58:18.928355 C  Upgrading to MythTV schema version 1334
2023-11-19 20:58:18.949653 C  Upgrading to MythTV schema version 1335
2023-11-19 20:58:18.971038 C  Upgrading to MythTV schema version 1336
2023-11-19 20:58:18.972236 C  Upgrading to MythTV schema version 1337
2023-11-19 20:58:18.991235 C  Upgrading to MythTV schema version 1338
2023-11-19 20:58:19.054146 C  Upgrading to MythTV schema version 1339
2023-11-19 20:58:19.093077 C  Upgrading to MythTV schema version 1341
2023-11-19 20:58:19.093519 C  Upgrading to MythTV schema version 1342
2023-11-19 20:58:19.094088 C  Upgrading to MythTV schema version 1343
2023-11-19 20:58:19.117273 C  Upgrading to MythTV schema version 1344
2023-11-19 20:58:19.138584 C  Upgrading to MythTV schema version 1345
2023-11-19 20:58:19.161498 C  Upgrading to MythTV schema version 1346
2023-11-19 20:58:19.185866 C  Upgrading to MythTV schema version 1347
2023-11-19 20:58:19.190686 C  Upgrading to MythTV schema version 1348
2023-11-19 20:58:19.285219 C  Upgrading to MythTV schema version 1349
2023-11-19 20:58:19.285950 C  Upgrading to MythTV schema version 1350
2023-11-19 20:58:19.286321 C  Upgrading to MythTV schema version 1351
2023-11-19 20:58:19.326582 C  Upgrading to MythTV schema version 1352
2023-11-19 20:58:19.427277 C  Upgrading to MythTV schema version 1353
2023-11-19 20:58:19.450006 C  Upgrading to MythTV schema version 1354
2023-11-19 20:58:19.475343 C  Upgrading to MythTV schema version 1355
2023-11-19 20:58:19.498903 C  Upgrading to MythTV schema version 1356
2023-11-19 20:58:19.500893 C  Upgrading to MythTV schema version 1357
2023-11-19 20:58:19.502625 C  Upgrading to MythTV schema version 1358
2023-11-19 20:58:19.503448 C  Upgrading to MythTV schema version 1359
2023-11-19 20:58:19.525097 C  Upgrading to MythTV schema version 1360
2023-11-19 20:58:19.526400 C  Upgrading to MythTV schema version 1362
2023-11-19 20:58:19.567529 C  Upgrading to MythTV schema version 1363
2023-11-19 20:58:19.599813 C  Upgrading to MythTV schema version 1365
2023-11-19 20:58:19.600784 C  Upgrading to MythTV schema version 1366
2023-11-19 20:58:19.619499 C  Upgrading to MythTV schema version 1367
2023-11-19 20:58:19.639185 C  Upgrading to MythTV schema version 1368
2023-11-19 20:58:19.663567 C  Upgrading to MythTV schema version 1369
2023-11-19 20:58:19.801188 C  Upgrading to MythTV schema version 1370
2023-11-19 20:58:19.900688 C  Upgrading to MythTV schema version 1371
2023-11-19 20:58:19.901698 C  Upgrading to MythTV schema version 1372
2023-11-19 20:58:19.922545 C  Upgrading to MythTV schema version 1373
2023-11-19 20:58:19.947490 C  Upgrading to MythTV schema version 1374
2023-11-19 20:58:19.951532 C  Upgrading to MythTV schema version 1375
2023-11-19 20:58:20.019889 C  Upgrading to MythTV schema version 1376
2023-11-19 20:58:20.020834 C  Upgrading to MythTV schema version 1377
2023-11-19 20:58:20.029585 C  Upgrading to MythTV schema version 1378
2023-11-19 20:58:20.029886 C  Upgrading to MythTV schema version 1379
No setting found for this machine's BackendServerAddr.
MythBackend starting in Web App only mode for initial setup.
Use http://<yourBackend>:6544 to perform setup.
2023-11-19 20:58:20.112650 I  Database schema upgrade complete.
2023-11-19 20:58:20.113379 I  Loading en_us translation for module mythfrontend
2023-11-19 20:58:20.116586 E  MythSocket(55f0b6da99e0:-1): Unable to lookup: 
2023-11-19 20:58:20.117965 N  ***********************************************************************
2023-11-19 20:58:20.117967 N  ***** MythBackend starting in Web App only mode for initial setup *****
2023-11-19 20:58:20.117968 N  ***** Use http://<yourBackend>:6544 to perform setup              *****
2023-11-19 20:58:20.117970 N  ***********************************************************************
2023-11-19 20:58:20.118087 I  HTTPPool: Using maximum 12 threads
2023-11-19 20:58:20.129574 W  SSL: SSL Host key file (/home/jim/.mythtv/certificates/key.pem) does not exist or is not readable
2023-11-19 20:58:20.134494 I  Listening on TCP 0.0.0.0:6544
2023-11-19 20:58:20.135376 I  Listening on TCP [::]:6544
2023-11-19 20:58:20.135398 I  Listening on TCP 0.0.0.0:6744
2023-11-19 20:58:20.135412 I  Listening on TCP [::]:6744
2023-11-19 20:58:21.090132 I  Bonjour: Service registration complete: name 'mythbackend on eos-mythtv' type '_http._tcp.' domain: 'local.'
2023-11-19 20:58:21.984593 I  Service '/services/' introspection complete
2023-11-19 20:58:21.984947 I  Service 'Myth' introspection complete
2023-11-19 20:58:21.994072 I  Service 'Config' introspection complete
2023-11-19 20:58:27.428260 I  Service 'Dvr' introspection complete
```
 Not sure if I'm not doing things in the right order or if the save button is not getting enabled for me to click on.  I'm just cycling through saving, restarting the backend, etc.  Finally I get past it.

**Comment by jfabernathy** [comment]
Sorry, this was an operator issue.  I redid the whole install EOS, yay -S mythtv-git --editmenu, and change to master.  Once mythtv was built, I fixed the mythtv user to be a standard user with /home/mythtv as home and added my user  as a member of group mythtv.

I installed mariadb and setup it up per the archlinux wiki for mariadb.

So at this point I opened a console and started mythbackend. (no systemd service yet)

In a browser I went to localhost:6544.  The order that worked was:
click test database.  It pops up the instructions for setting up mythconverg. I selected mariadb and created the script as instructed.  I then setup mythconverg importing the script. 

The instructions clearly say to click save and restart the backend.  So this time I clicked save and rolled back up and click restart backend.  This time it worked as expected and I could complete setup.

Sorry for the noise.
